cookbook/remote_access/debugging_workflows_tasks.py

```diff
@@ -25,4 +25,42 @@
 
 So here the investigation can move ahead by describing the pod and checking the issue with Image pull.
 
+Using flytekit (python):
+========================
+For how to inspect an execution, go back to `Inspecting Workflow and Task Executions <https://docs.flyte.org/projects/cookbook/en/latest/auto/remote_access/inspecting_executions.html>`__
+
+More details can be found in the docs for  `FlyteRemote <https://docs.flyte.org/projects/flytekit/en/latest/remote.html>`__
+
+.. code-block:: python
+        from flytekit.remote import FlyteRemote
+
+        # FlyteRemote object is the main entrypoint to API
+        remote = FlyteRemote(
+                config=Config.for_endpoint(endpoint="flyte.example.net"),
+                default_project="flytesnacks",
+                default_domain="development",
+        )
+        
+        # Fetch execution
+        execution = remote.fetch_execution(
+                name="fb22e306a0d91e1c6000", project="flytesnacks", domain="development"
+        )
+
+        # You can use FlyteRemote.sync() to to sync the entity object's state with the remote state during the execution run
+        synced_execution = remote.sync(execution, sync_nodes=True)
+        node_keys = synced_execution.node_executions.keys()
```

Comment:
Can you add sync to the inspecting executions page and remove all changes to this file?